platform/stacks/portal.stack.yml

```diff
@@ -0,0 +1,25 @@
+version: "3"
+
+networks:
+  default:
+    external:
+      name: ampnet
+  infrastructure:
+
+services:
+
+  portal:
+    image: appcelerator/amp-ui:1.0.0
```

Comment:
we started aligning the versions of the artifacts. Please use the "${TAG:-0.10.1}" version as for other core services.